File: src/main/java/gr/apt/persistence/enumeration/Job.java
package gr.apt.persistence.enumeration;

import java.util.List;

public enum Job {
    PROGRAMMER,
    PROJECT_MANAGER,
    SECRETARY,
    ACCOUNTANT;

    public static List<String> getValuesList(){
        return List.of(Job.PROGRAMMER.toString(),
                Job.PROJECT_MANAGER.toString(),
                Job.SECRETARY.toString(),
                Job.ACCOUNTANT.toString());
    }
}

File: src/main/java/gr/apt/lms/persistence/enumeration/Job.kt
package gr.apt.lms.persistence.enumeration

enum class Job {
    PROGRAMMER, PROJECT_MANAGER, SECRETARY, ACCOUNTANT;

    companion object {
        @JvmStatic
        val valuesList: List<String>
            get() = listOf(
                PROGRAMMER.toString(),
                PROJECT_MANAGER.toString(),
                SECRETARY.toString(),
                ACCOUNTANT.toString()
            )
    }
}